### PR TITLE
Portfolio Form Validation Project

### PR DESCRIPTION
**Pull request by anyars-encarta**
In this project,

- [ ] added form validation using JavaScript
- [ ] A message is displayed near the submit button if name is empty
- [ ] A message is displayed near the submit button if email is empty
- [ ] A message is displayed near the submit button if email is typed in UPPERCASE is empty
- [ ] In each error case, the form is not submitted

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -9,4 +9,121 @@ hamburger.addEventListener('click', () => {
...
+    e.preventDefault();
+    errorElement.innerText = 'Please enter your name';
+  } else if (email.value === '' || email.value == null) {
+    e.preventDefault();
+    errorElement.innerText = 'Please enter email';
+  } else if (email.value === email.value.toUpperCase()) {
+    e.preventDefault();
+    errorElement.innerText = 'Please use Lowercase';
+  }
+});
```
Comment (usorfaitheloho) [inline app.js]: You did well with your project, however your validation seems to not be working properly. when a user inputs an email address with a mixture of uppercase and lowercase letters, the form seems to submit, however when a user uses only uppercase letters, it doesn't submit. Please ,according to the project requirement, the form is supposed to check for validation even if it has just one uppercase letter 

Project requirement : 
image

Your validation :

https://github.com/anyars-encarta/anyars-encarta.github.io/assets/20537712/45758bb3-f5b5-4c87-bd55-027b29ba2a50

File: index.html [hunk 2]
```diff
@@ -293,14 +295,15 @@ <h4 class="interest-text">I'm always interested in hearing about new projects, s
...
+            <input type="text" class="full-name" id="fullname" name="name" placeholder="Full name" maxlength="30">
             <hr class="end-name">
             <label id="mylabel" for="email-address" hidden>Email Address</label>
-            <input type="email" class="email-address" id="email-address" name="email" placeholder="Email address" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required>
+            <input type="email" class="email-address" id="email-address" name="email" placeholder="Email address">
             <hr class="end-address">
             <label id="mylabel" for="write-message" hidden>Write something here</label>
-            <textarea class="text-area" id="write-message" name="message" rows="7" cols="50" maxlength="500" placeholder="Write me something..." required></textarea>
+            <textarea class="text-area" id="write-message" name="message" rows="7" cols="50" maxlength="500" placeholder="Write me something..."></textarea>
             <button class="form-button" type="submit">Get in touch</button>
```
Comment (usorfaitheloho) [inline index.html]: [Optional] Please add  `required` for all your input values so as to enable a user fill all input values before submitting a form 😄

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @anyars-encarta  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * contact form validation is almost completed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by skyv26 — APPROVED** [review]
## Hi @anyars-encarta, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Linters are green ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**